src/main/resources/hudson/plugins/deploy/tomcat/TomcatAdapter/config.jelly

```diff
@@ -8,6 +8,9 @@
     <f:textbox />
   </f:entry>
   <f:advanced>
+    <f:entry title="Alternative deployment context" field="alternativeDeploymentContext">
```

Comment:
Help file is missing (see https://github.com/jenkinsci/deploy-plugin/blob/master/src/main/resources/hudson/plugins/deploy/tomcat/TomcatAdapter/help-path.html for an example).